client/my-sites/sidebar/test/sidebar.js

```diff
@@ -126,11 +118,10 @@ describe( 'MySitesSidebar', () => {
 			expect( wrapper.html() ).toEqual( null );
 		} );
 
-		test( "Should return null if user who can't upgrade user can not use store on this site (control a/b group)", () => {
-			abtest.mockImplementation( () => 'control' );
+		test( "Should return null if user who can't upgrade user can not use store on this site", () => {
 			const Sidebar = new MySitesSidebar( {
 				canUserUseCalypsoStore: false,
-				canUserUpgradeSite: true,
+				canUserUpgradeSite: false,
```

Comment:
👍  Good eye!

Reply:
I couldn't find a difference between these two tests, and figured there is a bug 🙂